utils/node_local_test.py

```diff
@@ -409,6 +409,29 @@ def stop(self):
 
         if not self._sp:
             return
+        rc = self.run_dmg(['system', 'stop'])
+        print(rc)
+
+        start = time.time()
+        while True:
+            time.sleep(0.5)
+            rc = self.run_dmg(['system', 'query'])
```

Comment:
FYI, I am changing this in my PR to use JSON output instead of doing string matching: https://github.com/daos-stack/daos/pull/4064/files#diff-74f1d143169d1655c88e727ed677f50cb035b054c6fc1ace2c615b3c91210725R392

We should probably pull the run_dmg()/json.loads() into a helper that appends the --json argument and parses the response.

Reply:
Worth knowing.  This code still uses the --recreate-superblocks option so does need updating.

I still can't get daos to stop though, it seems to return "Stopping" continuously and never progress to "Stopped" so I almost left out this part of the PR.

Reply:
Hmm. I think we may have a regression here that wasn't noticed because we haven't been using stop without --force. @tanabarr, please take a look at this. I tried Ashley's PR on master and on your latest events branch and in both cases the second instance on my system never moved from stopping->stopped. I can also see that there is still a daos_io_server process running.

Reply:
Without the other change in this PR daos_io_server would probably segfault, and therefore wouldn't be running afterwards, although I doubt that's how we intend it to work.

Reply:
the use of stop without force normally results in errored state but this PR catches the segfault as Ashley has mentioned, the behaviour you are describing is an artifact of suppressing that. happy to look into it further if you could raise a ticket and assign to me. I've worked with Ashley and we've got this PR working now.